src/style.css

```diff
@@ -1 +1,29 @@
-/* add styles here */
+.page {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin: 0;
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style elements using `.classes`. For example, instead of styling `.stars`, use a class that describes the function or content of the element, like `.rating` or `.review-stars`.